bin/cohere-embed-v3-services.ts

```diff
@@ -23,6 +23,7 @@ checkEnvVariables('APP_NAME',
     'DATA_INGESTION_API_KEY',
     'CPU_TYPE',
     'MEMORY_TYPE',
+    'VPC_ID',
 );
 
 const appName = process.env.APP_NAME!;
```

Comment:
**Suggestion:** The `process.env` variables are being accessed with non-null assertions (`!`). This can lead to runtime errors if the environment variables are not set. Consider adding a check to ensure these variables are defined before using them.
> 📌  **Medium Priority** 
 > [possible issue, importance: 7]